xinference/model/llm/pytorch/baichuan.py

```diff
@@ -54,6 +57,37 @@ def _load_model(self, **kwargs):
 
             raise ImportError(f"{error_message}\n\n{''.join(installation_guide)}")
 
+        # load model from tensorizer
+        enable_tensorizer = self._pytorch_model_config.get("enable_tensorizer", None)
+        if enable_tensorizer:
+            from .tensorizer_utils import (
+                check_tensorizer_integrity,
+                load_from_tensorizer,
+            )
+
+            component_types = [("tokenizer", AutoTokenizer)]
+            model_prefix = "model"
+            if not check_tensorizer_integrity(
+                self.model_path,
+                model_prefix,
+                [component[0] for component in component_types],
+            ):
```

Comment:
Should these codes be a util function for all the models. Otherwise there are too many duplicated codes.